### PR TITLE
changing icons from png/gif to svg

### PR DESCRIPTION
**Pull request by yayassssine**
## Description

changing all icons from format png to new files svg

**Fixes** #MON-12612

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [x] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 20.10.x
- [ ] 21.04.x
- [ ] 21.10.x
- [x] 22.04.x (master)

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

Any **relevant details** of the configuration to perform the test should be added.

## Checklist

- [x] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: www/Themes/Generic-theme/Variables-css/variables.css [hunk 1]
```diff
@@ -192,6 +192,7 @@
     --badge-downtime-background-color : var(--color-mauve);
     --badge-undetermined :var(--color-iron);
     --status-success-color:var(--color-white);
+    --ico-16-all-services:var(--color-primary-dark);
```
Comment (kduret) [inline www/Themes/Generic-theme/Variables-css/variables.css:195]: move it after `--ico-14-fill-color` to make it consistent with light theme

File: www/include/common/templates/cloneHost.ihtml [hunk 2]
```diff
@@ -29,10 +29,16 @@
                     <img src="./img/icons/move.png" class="clonehandle ico-14" style="vertical-align:middle;">
                 </span>
             {/if}
-                <img src='./img/icons/edit_mode.png' title="Edit template" class="ico-14" style='cursor:pointer;vertical-align:middle;' onclick="editHostTemplate(this)">
+            <svg xmlns="http://www.w3.org/2000/svg" title="Edit template" class="ico-14" style='cursor:pointer;vertical-align:middle;' onclick="editHostTemplate(this)" viewBox="0 0 24 24">
+            <path d="M0 0h24v24H0z" fill="none"/>
+            <path d="M3 17.25V21h3.75L17.81 9.94l-3.75-3.75L3 17.25zM20.71 7.04c.39-.39.39-1.02 0-1.41l-2.34-2.34c-.39-.39-1.02-.39-1.41 0l-1.83 1.83 3.75 3.75 1.83-1.83z"/>
```
Comment (kduret) [inline www/include/common/templates/cloneHost.ihtml:34]:
```suggestion
                <path d="M0 0h24v24H0z" fill="none"/>
                <path d="M3 17.25V21h3.75L17.81 9.94l-3.75-3.75L3 17.25zM20.71 7.04c.39-.39.39-1.02 0-1.41l-2.34-2.34c-.39-.39-1.02-.39-1.41 0l-1.83 1.83 3.75 3.75 1.83-1.83z"/>
```

File: www/include/common/templates/cloneMacro.ihtml [hunk 3]
```diff
@@ -39,7 +39,10 @@
                 <img src="./img/icons/move.png" class="ico-14" style="vertical-align:middle;">
             </span>
             <span id="{$cloneId}_remove_current" style=''>
-                <img src="./img/icons/circle-cross.png" class='ico-14' style="vertical-align:middle;">
+            <svg xmlns="http://www.w3.org/2000/svg"  class="ico-14" style="vertical-align:middle;" viewBox="0 0 24 24">
+                                            <path d="M0 0h24v24H0z" fill="none"/>
+                                            <path d="M12 2C6.47 2 2 6.47 2 12s4.47 10 10 10 10-4.47 10-10S17.53 2 12 2zm5 13.59L15.59 17 12 13.41 8.41 17 7 15.59 10.59 12 7 8.41 8.41 7 12 10.59 15.59 7 17 8.41 13.41 12 17 15.59z"/>
+                                        </svg>
```
Comment (kduret) [inline www/include/common/templates/cloneMacro.ihtml:45]: fix indent

File: www/include/configuration/configObject/host/listHost.ihtml [hunk 4]
```diff
@@ -64,8 +64,12 @@
...
+</g></g></svg>                </a>
```
Comment (kduret) [inline www/include/configuration/configObject/host/listHost.ihtml:72]: fix indent

File: www/include/configuration/configObject/service/formService.ihtml [hunk 5]
```diff
@@ -71,7 +71,10 @@
...
+                         &nbsp;<svg xmlns="http://www.w3.org/2000/svg"  title="Edit template" class="ico-14" style='cursor:pointer;vertical-align:middle;' onClick="window.open('main.php?p=60206&service_id='+ document.Form.elements['service_template_model_stm_id'].options[document.Form.elements['service_template_model_stm_id'].selectedIndex].value + '&o=c&min=1','','toolbar=no,location=no,directories=no,status=no,scrollbars=yes,resizable=yes,copyhistory=no, width=800, height=600');" viewBox="0 0 24 24">
+                                <path d="M0 0h24v24H0z" fill="none"/>
+                                <path d="M3 17.25V21h3.75L17.81 9.94l-3.75-3.75L3 17.25zM20.71 7.04c.39-.39.39-1.02 0-1.41l-2.34-2.34c-.39-.39-1.02-.39-1.41 0l-1.83 1.83 3.75 3.75 1.83-1.83z"/>
+                                </svg>
```
Comment (kduret) [inline www/include/configuration/configObject/service/formService.ihtml:77]: fix indent

File: www/include/home/customViews/index.ihtml [hunk 6]
```diff
@@ -2,7 +2,10 @@
 <div id='globalView'>
     {if $aclParameters == 1}
     <div class="toggleEdit">
-        <a href="#" title="{$editMode}"><img src="./img/icons/edit_mode.png" class="ico-14"></a>
+        <a href="#" title="{$editMode}"><svg xmlns="http://www.w3.org/2000/svg" class="ico-14" viewBox="0 0 24 24">
+                                        <path d="M0 0h24v24H0z" fill="none"/>
+                                        <path d="M3 17.25V21h3.75L17.81 9.94l-3.75-3.75L3 17.25zM20.71 7.04c.39-.39.39-1.02 0-1.41l-2.34-2.34c-.39-.39-1.02-.39-1.41 0l-1.83 1.83 3.75 3.75 1.83-1.83z"/>
+                                        </svg></a>
     </div>
```
Comment (kduret) [inline www/include/home/customViews/index.ihtml:9]: fix indent

File: www/include/monitoring/status/HostGroups/templates/hostGroup.ihtml [hunk 7]
```diff
@@ -28,11 +28,22 @@
...
+    <path d="M10 16.5l6-4.5-6-4.5v9zM12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8z"/>
+</svg></span>
                     <span class="consol_button"><a class="" id="JS_monitoring_pause" href="#" onclick="javascript:monitoring_pause('');"><img src='./img/icons/media_pause.png' alt='Pause' title='Pause'></a></span>
-                    <span class="consol_button"><img id="JS_monitoring_pause_gray" class="cachediv" src='./img/icons/media_pause_gray.png' alt='Pause' title='Pause'></span>
+                    <span class="consol_button">
+                    <svg xmlns="http://www.w3.org/2000/svg" class="cachediv"  alt='Pause' title='Pause' viewBox="0 0 24 24">
+                    <path d="M0 0h24v24H0z" fill="none"/>
+                    <path d="M9 16h2V8H9v8zm3-14C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8zm1-4h2V8h-2v8z"/>
+                    </svg>
+                    </span>
```
Comment (kduret) [inline www/include/monitoring/status/HostGroups/templates/hostGroup.ihtml:46]: fix indent

File: www/include/common/javascript/jquery/plugins/select2/js/select2.min.js [hunk 8]
```diff
@@ -1,3 +1,2392 @@
...
\ No newline at end of file
+/*! Select2 4.0.3 | https://github.com/select2/select2/blob/master/LICENSE.md */
```
Comment (kduret) [inline www/include/common/javascript/jquery/plugins/select2/js/select2.min.js:1]: js file is not minified anymore

File: www/include/common/javascript/centreon/brokerConfig.js [hunk 9]
```diff
@@ -39,8 +39,11 @@ function clonifyTableFields(attributeName,displayName){
         var table2 = jQuery('<table>').addClass('formTable table').css({'position': 'relative','border': '1px solid #e7e7e8'});
         var clone_template = jQuery('<tr>').attr('id',obj +"_template" ).addClass('clone_template').append(jQuery('<td>').attr('colspan','2').css({'padding': '0 12px'}).append(table2));
         table1.append(clone_template).append(jQuery('<tr>').attr('id',obj +"_noforms_template" ));
-        var img = jQuery('<img>').attr('src','./img/icons/circle-cross.png').addClass('ico-14').css('vertical-align','middle');
-        var remove = jQuery('<span>').css({'cursor': 'pointer', 'position': 'absolute', 'top': '56px', 'right': '-17px'}).attr('id', obj+'_remove_current').append(img);
+        var svg = jQuery('<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"  class="ico-14-circle-cross">\n' +
```
Comment (callapa) [inline www/include/common/javascript/centreon/brokerConfig.js:42]: Use the SVG file instead

File: www/include/common/javascript/centreon/centreon-select2-optgroup.js [hunk 10]
```diff
@@ -122,10 +122,10 @@
...
           .addClass('clearAllSelect2')
           .attr('title', this.translate('Clear field'))
-          .append(
-              $('<img>')
-                  .attr('src', './img/icons/circle-cross.png')
-                  .addClass('ico-14')
+          .append('<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"  class="ico-14-circle-cross">\n' +
+              '<path d="M0 0h24v24H0z" fill="none"/>\n' +
+              '<path d="M14.59 8L12 10.59 9.41 8 8 9.41 10.59 12 8 14.59 9.41 16 12 13.41 14.59 16 16 14.59 13.41 12 16 9.41 14.59 8zM12 2C6.47 2 2 6.47 2 12s4.47 10 10 10 10-4.47 10-10S17.53 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8z"/>\n' +
+              '</svg>'
```
Comment (callapa) [inline www/include/common/javascript/centreon/centreon-select2-optgroup.js:128]: Use the SVG file instead

File: www/include/common/javascript/centreon/centreon-select2-optgroup.js [hunk 11]
```diff
@@ -366,7 +366,10 @@
       var $closeButton = $('<a>')
           .addClass('close')
           .css('cursor', 'pointer')
-          .append('<img src="./img/icons/circle-cross.png" class="ico-18">');
+          .append('<svg xmlns="http://www.w3.org/2000/svg" class="ico-18"  style="vertical-align:middle;" viewBox="0 0 24 24">\n' +
+              '                    <path d="M0 0h24v24H0z" fill="none"/>\n' +
+              '                    <path d="M12 2C6.47 2 2 6.47 2 12s4.47 10 10 10 10-4.47 10-10S17.53 2 12 2zm5 13.59L15.59 17 12 13.41 8.41 17 7 15.59 10.59 12 7 8.41 8.41 7 12 10.59 15.59 7 17 8.41 13.41 12 17 15.59z"/>\n' +
+              '                </svg>');
```
Comment (callapa) [inline www/include/common/javascript/centreon/centreon-select2-optgroup.js:372]: Use the SVG file instead

File: www/include/common/javascript/centreon/centreon-select2.js [hunk 12]
```diff
@@ -122,11 +122,10 @@
...
           .attr('title', this.translate('Clear field'))
-          .append(
-              $('<img>')
-                  .attr('src', './img/icons/circle-cross.png')
-                  .addClass('ico-14')
-          );
+          .append('<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"  class="ico-14-circle-cross">\n' +
+              '<path d="M0 0h24v24H0z" fill="none"/>\n' +
+              '<path d="M14.59 8L12 10.59 9.41 8 8 9.41 10.59 12 8 14.59 9.41 16 12 13.41 14.59 16 16 14.59 13.41 12 16 9.41 14.59 8zM12 2C6.47 2 2 6.47 2 12s4.47 10 10 10 10-4.47 10-10S17.53 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8z"/>\n' +
+              '</svg>');
```
Comment (callapa) [inline www/include/common/javascript/centreon/centreon-select2.js:128]: Use the SVG file instead

File: www/include/common/javascript/centreon/centreon-select2.js [hunk 13]
```diff
@@ -385,7 +384,10 @@
       var $closeButton = $('<a>')
           .addClass('close')
           .css('cursor', 'pointer')
-          .append('<img src="./img/icons/circle-cross.png" class="ico-18">');
+          .append('<svg xmlns="http://www.w3.org/2000/svg" class="ico-18"  style="vertical-align:middle;" viewBox="0 0 24 24">\n' +
+              '                    <path d="M0 0h24v24H0z" fill="none"/>\n' +
+              '                    <path d="M12 2C6.47 2 2 6.47 2 12s4.47 10 10 10 10-4.47 10-10S17.53 2 12 2zm5 13.59L15.59 17 12 13.41 8.41 17 7 15.59 10.59 12 7 8.41 8.41 7 12 10.59 15.59 7 17 8.41 13.41 12 17 15.59z"/>\n' +
+              '                </svg>');
```
Comment (callapa) [inline www/include/common/javascript/centreon/centreon-select2.js:390]: Use the SVG file instead

File: www/include/common/javascript/centreon/popin.js [hunk 14]
```diff
@@ -38,7 +38,10 @@
   function CentreonPopin(settings, $elem) {
     var self = this,
         closeBtn = $('<a class="close" href="#">' +
-                    '<img src="./img/icons/circle-cross.png" class="ico-18"></a>'),
+                    '                <svg xmlns="http://www.w3.org/2000/svg"  class="ico-18" style="vertical-align:middle;" viewBox="0 0 24 24">\n' +
+            '                    <path d="M0 0h24v24H0z" fill="none"/>\n' +
+            '                    <path d="M12 2C6.47 2 2 6.47 2 12s4.47 10 10 10 10-4.47 10-10S17.53 2 12 2zm5 13.59L15.59 17 12 13.41 8.41 17 7 15.59 10.59 12 7 8.41 8.41 7 12 10.59 15.59 7 17 8.41 13.41 12 17 15.59z"/>\n' +
+            '                </svg></a>'),
```
Comment (callapa) [inline www/include/common/javascript/centreon/popin.js:44]: Use the SVG file instead

File: www/include/common/templates/clone.ihtml [hunk 15]
```diff
@@ -29,7 +29,10 @@
                     <img src="./img/icons/move.png" class="clonehandle ico-14" style="vertical-align:middle;">
                 </span>
                 <a id="{$cloneId}_remove_current" style="margin-right: 4px;">
-                    <img src="./img/icons/circle-cross.png" class="ico-14" style="vertical-align:middle;">
+                <svg xmlns="http://www.w3.org/2000/svg" class="ico-14" style="vertical-align:middle;" viewBox="0 0 24 24">
+                    <path d="M0 0h24v24H0z" fill="none"/>
+                    <path d="M12 2C6.47 2 2 6.47 2 12s4.47 10 10 10 10-4.47 10-10S17.53 2 12 2zm5 13.59L15.59 17 12 13.41 8.41 17 7 15.59 10.59 12 7 8.41 8.41 7 12 10.59 15.59 7 17 8.41 13.41 12 17 15.59z"/>
+                </svg>
```
Comment (callapa) [inline www/include/common/templates/clone.ihtml:35]: Use the SVG file instead

**Review by callapa — CHANGES_REQUESTED** [review]
Use the SVG file instead wherever possible

**Pull request closed without merging**